src/qtgui/plotter.cpp

```diff
@@ -1174,7 +1174,7 @@ void CPlotter::getScreenIntegerFFTData(qint32 plotHeight, qint32 plotWidth,
         for (i = minbin; i < maxbin; i++)
             m_pTranslateTbl[i] = ((qint64)(i-m_BinMin)*plotWidth) / (m_BinMax - m_BinMin);
         *xmin = m_pTranslateTbl[minbin];
-        *xmax = m_pTranslateTbl[maxbin - 1];
```

Comment:
`xmax` is supposed to be the first unused column, not the last used column. The off-by-one prevents the rightmost column from being used in the `largeFft` case. (It's used once you zoom in and reach the `else` case below, which returns the correct `xmax` value.)